internal/infrastructure/router/public_test.go

```diff
@@ -152,13 +153,24 @@ func TestNewGroupPublic(t *testing.T) {
 
 		// This routes are added when the group is created
 		appPrefix + appName + versionFull + "/*": {
-			"echo_route_not_found": "github.com/labstack/echo/v4.glob..func1",
+			"echo_route_not_found": "github.com/labstack/echo/v4.init.func1",
```

Comment:
What is the reason for this test to be testing the handlers? I don't see the benefit.  From my PoV it is looking at too much details and it is not really testing any behavior.  It would be good enough to check that `e.Routes` contains all the expected routes with correct names and http methods. 

That way `TestNewGroupPublic` test would be simpler and would not rely on internal implementation of echo framework.

Reply:
thanks! I have updated the code to enhance the unit test and get rid of that specific details.